console/src/components/Services/Metadata/Actions.js

```diff
@@ -118,6 +138,71 @@ export const loadInconsistentObjects = (
   };
 };
 
+/* Reloads only remote schema metadata */
+
+export const reloadRemoteSchema = (remoteSchemaName, successCb, failureCb) => {
+  return (dispatch, getState) => {
+    const headers = getState().tables.dataHeaders;
+    const { featuresCompatibility } = getState().main;
+    let reloadQuery = {};
+
+    if (
+      RELOAD_METADATA_API_CHANGE in featuresCompatibility &&
+      featuresCompatibility[RELOAD_METADATA_API_CHANGE]
+    ) {
+      reloadQuery = {
+        ...reloadRemoteCacheAndGetInconsistentObjectsQuery(remoteSchemaName),
+      };
+    } else {
+      reloadQuery = {
+        ...reloadCacheAndGetInconsistentObjectsQuery,
+      };
+    }
+    dispatch({ type: LOADING_METADATA });
+    return dispatch(
+      requestAction(endpoints.query, {
+        method: 'POST',
+        headers,
+        body: JSON.stringify(reloadQuery),
+      })
+    ).then(
+      data => {
+        const allSchemas = getState().tables.allSchemas;
+        const functions = getState().tables.trackedFunctions;
+        const inconsistentObjects = data[1].inconsistent_objects;
+        dispatch({
```

Comment:
The next 30 lines are an exact copy-paste of the `loadInconsistentObjects` function defined above. Pull out common code